.github/workflows/calibreapp-image-actions.yml

```
@@ -0,0 +1,22 @@
+name: Compress Images
+on:
+  pull_request:
+    paths:
+      - '**.jpg'
+      - '**.jpeg'
+      - '**.png'
+      - '**.webp'
+jobs:
+  build:
+    # Only run on Pull Requests within the same repository, and not from forks.
+    # if: github.event.pull_request.head.repo.full_name == github.repository
+    name: calibreapp/image-actions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```

https://github.com/actions/checkout